src/main/java/com/minecolonies/coremod/colony/buildings/BuildingBaker.java

```diff
@@ -55,28 +55,28 @@ public int getMaxBuildingLevel()
     }
 
     /**
-     * The name of the baker's job.
+     * Create a Baker job.
      *
-     * @return The name of the baker's job.
+     * @param citizen the citizen to take the job.
+     * @return The new Baker job.
      */
     @NotNull
     @Override
-    public String getJobName()
+    public AbstractJob createJob(final CitizenData citizen)
     {
-        return BAKER;
+        return new JobPlaceholder(citizen); //TODO Implement Later
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ATrailingCommentCheck)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)